src/migrations.ts

```diff
@@ -210,13 +236,9 @@ export class Migration {
    * Returns next upgrade script object
    */
   private async getNextUpgradeScript(): Promise<MigrationScript> {
-    const { version } = await this.getLastVersion();
-    for (const script of this.scripts) {
-      if (script.version === (version + 1)) {
-        return script;
-      }
-    }
-    return null;
+    const loadedScripts = await this.getLoadedScripts();
+    const firstNotLoadedScript = this.scripts.find((script) => !loadedScripts.find((loadedScripts) => loadedScripts.version === script.version));
```

Comment:
Here is a confusing variable. Use `loaded` instead of `loadedScripts` for most inner find function.